data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/beaconhandlers/BeaconValidatorsHandler.java

```diff
@@ -111,15 +117,10 @@ public void handle(Context ctx) throws Exception {
               state -> {
                 if (state.isEmpty()) {
                   // empty list
-                  return jsonProvider.objectToJSON(
-                      produceResponse(new SSZList<>(Validator.class, 0L), pageSize, pageToken));
-                }
-                if (activeOnly) {
-                  return jsonProvider.objectToJSON(
-                      produceResponse(state.get().getActiveValidators(), pageSize, pageToken));
+                  return jsonProvider.objectToJSON(produceEmptyListResponse());
```

Comment:
nit: because it simplifies to array in json, there really is no difference between wrapping this in a new object return, passing List.of(), or passing "[]"
I'd tend toward getting rid of the wrapper function and just returning jsonProvider.objectToJSON(list.of())...